app/forms/sufia/forms/collection_form.rb

```diff
@@ -1,6 +1,4 @@
 module Sufia::Forms
```

Comment:
I tried nuking the entire file for both files that no longer have any code and got constant missing errors for both.  I can look further into why the constant missing errors are occurring.